index.md

```diff
@@ -1,12 +1,45 @@
-# Welcome!
+# Welcome & Congratulations!
+### You just took your first step towards becoming a Virtual Intern!
 
-This MDwiki serves as an information repository for treehouses, a project of the Open Learning Exchange.  Please feel free to join us on our [Gitter chat](https://gitter.im/treehouses/Lobby).
+<br />
```

Comment:
@sonAndrew please do not use HTML

Reply:
Why not? I only used it for spacing and one image.